_posts/2021-03-18-gui-react-patternfly.adoc

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: Adding Eyecandy to your Quarkus App with React and Patternfly 
+date: 2021-03-18
```

Comment:
is that your intended publish date?

Reply:
It was just a random date in the future. Feel free to move it to whenever it fits.